### PR TITLE
Fix failing CI tests

### PR DESCRIPTION
**Pull request by nbierdeman**
### Purpose

This PR fixes the failing CI tests.

### CODE REVIEW

File: spec/client/javascript/standards/components.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ paypal.Component({
 
 ```javascript
 try {
-    const component = await paypal.Component({
+    const component = /* await */ paypal.Component({
```
Comment (crookedneighbor) [inline spec/client/javascript/standards/components.md:107]: For these, is it angry that it's not in an async function?

Comment (nbierdeman) [reply spec/client/javascript/standards/components.md:107]: I’m guessing that’s the case. I wonder if `operation()` and `Component()` would need to be declared w/ async? I can give that a try.

Comment (nbierdeman) [reply spec/client/javascript/standards/components.md:107]: Oh, yeah, `operation()` and `Component()` are defined on the `paypal` object. 🤔

**Comment by leogedler** [comment]
LGTM

**Review by leogedler — APPROVED** [review]
LGTM

**Comment by nbierdeman** [comment]
@crookedneighbor, I implemented your TS ignore errors suggestion, and it solved the `await` errors when testing locally. Assuming CI passes, I think this is ready for your final review.

File: spec/client/javascript/callbacks/createSubscription.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This callback will be invoked whenever the user takes an action which requires a
 ```javascript
 const createSubscription = (data, actions) => {
     return actions.subscription.create({
-        __TBD__
+        __TBD__: () => { /* logic here */ }
```
Comment (crookedneighbor) [inline spec/client/javascript/callbacks/createSubscription.md:16]:
```suggestion
        // TBD
```

**Review by crookedneighbor — CHANGES_REQUESTED** [review]
This is looking good. Just had one change I want to see around the `__TBD__` bit, then it's good to merge.

File: spec/client/javascript/callbacks/createSubscription.md (hunk 2 above)
Comment (nbierdeman) [reply spec/client/javascript/callbacks/createSubscription.md]: @crookedneighbor, a value is required for the tests to pass, so I set it to the string `'TBD'`. Are you cool with that?

Comment (crookedneighbor) [reply spec/client/javascript/callbacks/createSubscription.md]: Yeah, that's fine

**Pull request merged**